packages/mui-material/src/Paper/Paper.js

```diff
@@ -107,6 +113,22 @@ const Paper = React.forwardRef(function Paper(inProps, ref) {
       className={clsx(classes.root, className)}
       ref={ref}
       {...other}
+      style={{
+        ...other.style,
```

Comment:
This should come last, props' style should override the predefined CSS vars if needed.